service/storage_service.cc

```diff
@@ -3563,7 +3563,7 @@ class node_ops_ctl {
             seastar::thread::maybe_yield();
             // FIXME: use node* rather than endpoint
             auto node = np->endpoint();
-            if (!ignore_nodes.contains(node) && sync_to_node(node)) {
+            if (!ignore_nodes.contains(node) && np->is_token_owner() && sync_to_node(node)) {
```

Comment:
The commit message still says is_token_owner_for_read

Reply:
fixed that (and the changelog message)